robolectric/src/main/java/org/robolectric/internal/ParallelUniverse.java

```diff
@@ -141,6 +173,11 @@ public void setUpApplicationState(Method method, TestLifecycle testLifecycle, Re
       appResources.updateConfiguration(configuration, appResources.getDisplayMetrics());
       shadowsAdapter.setAssetsQualifiers(appResources.getAssets(), qualifiers);
 
+      if (Integer.valueOf(qualifiers.substring(1)) >= 21) {
+        final String SDK = System.getenv("ANDROID_HOME");
+        File f = new File(SDK + "/platforms/android-21");
+        RenderServiceFactory factory = RenderServiceFactory.create(f);
```

Comment:
Initialize the resource management used by BridgeContext and all the delegate classes. E.g. without this, ShadowTextPaintTest will crash because when trying to init a TextPaint, TextPaint_Delegate will throw ResourceNotFoundError.